pkg/loader/executor.go

```diff
@@ -302,15 +358,28 @@ func tryRefreshTableErr(err error) bool {
 	return false
 }
 
+func tryRefreshTableOracleErr(err error) bool {
+	oraErr, ok := godror.AsOraErr(err)
+	if !ok {
+		return false
+	}
+	//Invalid identifier for oracle db error
```

Comment:
Is there any link for this code?
```suggestion
	// Invalid identifier for oracle db error
```